docs/production-deployment/cloud/rto-rpo.mdx

```diff
@@ -19,75 +19,99 @@ tags:
+    1. Namespaces without Multi-cloud Replication: 8-hour RPO* and Unbounded RTO (dependent on how long the cloud provider takes to recover)
+    2. Namespaces with Multi-cloud Replication and Temporal-initiated failovers enabled*: 1-minute RPO and 20 minutes or less RTO
 
-As Workflows progress in the active region, history events are asynchronously replicated to the standby region.
-In case of an incident or outage in the active region, Temporal Cloud will fail over to the standby region so that existing Workflow Executions will continue to run and new Executions can be started.
+A few notes on these goals:
 
-**Recovery Point Objective (RPO) - sub-1-minute**
+- "8-hour RPO" for Namespaces without the appropriate High Availability feature: Historically, regional outages have not led to data corruption or permanent data loss in data systems that are replicated across three availability zones. All Namespace data was available once the outage ended; affected Namespaces observed a recovery point of "zero" (no data loss). However, as a precaution, Temporal backs up all Namespaces in rolling 4-hour windows. Should a future outage cause permanent data loss in the underlying data system, these backups would meet an 8-hour RPO.
 
-Temporal Cloud is designed to limit data loss after recovery when the incident triggering the failover is resolved.
+- "Temporal-initiated failovers:" Also known as "automatic failovers," these failovers are initiated by Temporal's tooling and/or on-call engineers on Namespaces that have High Availability enabled. **Temporal highly recommends keeping Temporal-initiated failovers enabled,** which is the default for all Namespaces with High Availability features. Users can still trigger manual failovers on their Namespaces even if Temporal-initiated failovers are enabled. When Temporal-initiated failovers are disabled on a Namespace, Temporal's RTO for that Namespace is unbounded (it is dependent on how long the underlying outage lasts)
 
-Temporal Cloud strives to maintain a P95 [replication lag](/cloud/high-availability/monitoring#replication-lag-metric) of less than 1 minute.
-In this context, P95 means 95% of updates are processed faster than this limit.
+## Minimizing the Recovery Point
 
-The recovery point objective RPO is near-zero.
-There may be a short period of time—the replication lag—during the incident when some data may be unavailable
+Temporal has put extensive work into tools and processes that minimize the recovery point and achieve its RPO for Temporal-initiated failovers, including:
 
-**Recovery Time Objective (RTO) - 20 minutes**
+- Best-in-class data replication technology that keeps the replica up to date with the active.
```

Comment:
Could link to https://www.youtube.com/watch?v=mULBvv83dYM where Liang gets into more specifics